### PR TITLE
fix(mongodb-old): add security context to correct section

### PR DESCRIPTION
**Pull request by TheRealDwright**
This moves the security context to alphabetical order, additionally
resolves error: mapping values are not allowed in this context when
attempting to render the mongo deployment chart

Ticket: BG-45014

### CODE REVIEW

**Comment by TheRealDwright** [comment]
Introduced by #25

**Review by james-callahan — APPROVED** [review]
I think the correct fix is:
```diff
diff --git a/stable/mongodb-old/templates/deployment.yaml b/stable/mongodb-old/templates/deployment.yaml
index a6acb3c2c..475a8b540 100755
--- a/stable/mongodb-old/templates/deployment.yaml
+++ b/stable/mongodb-old/templates/deployment.yaml
@@ -72,7 +72,7 @@ spec:
           claimName: {{ template "fullname" . }}
       {{- else }}
         emptyDir: {}
-      {{- end -}}
+      {{- end }}
       securityContext:
         fsGroup: 1001
         runAsUser: 1001
```

but yours works too

File: stable/mongodb-old/templates/deployment.yaml [hunk 1]
```diff
@@ -65,14 +65,14 @@ spec:
...
       {{- if .Values.persistence.enabled }}
         persistentVolumeClaim:
           claimName: {{ template "fullname" . }}
       {{- else }}
         emptyDir: {}
-      {{- end -}}
-      securityContext:
-        fsGroup: 1001
-        runAsUser: 1001
+      {{- end}}
```
Comment (james-callahan) [inline stable/mongodb-old/templates/deployment.yaml:78]:
```suggestion
      {{- end }}
```

**Review by james-callahan — APPROVED** [review]

File: stable/mongodb-old/templates/deployment.yaml (hunk 1 above)
Comment (TheRealDwright) [reply stable/mongodb-old/templates/deployment.yaml:78]: Opted for consistency with the rest of the repo.

**Pull request merged**